### DISCUSSION TITLE
SFML docset

### DISCUSSION THREAD

**Discussion by Napolus** [zealdocs/zeal#1334] [Q&A] [answered]
please add SFML documentation

**Comment by richarson** [comment]
Zeal doesn't provide docsets, it uses Dash's.

See https://github.com/zealdocs/zeal/issues/1312#issuecomment-766183527 for more information.

**Comment by Layerella** [comment]
zeal can not download Docset.why,I am chinese

**Comment by trollixx** [comment, marked as answer]
Docsets are not part of this projects. All docsets currently available inside Zeal are generously provided by [Dash](https://kapeli.com/dash).

Looks like someone built a docset for SFML already: https://github.com/tsoding/sfml-docset
